shared/naturalcrit/markdown.js

```diff
@@ -328,14 +327,21 @@ const voidTags = new Set([
 const processStyleTags = (string)=>{
 	//split tags up. quotes can only occur right after colons.
 	//TODO: can we simplify to just split on commas?
-	const tags = string.match(/(?:[^, ":]+|:(?:"[^"]*"|))+/g);
+	const tags = string.match(/(?:[^,":=]+|[:=](?:"[^"]*"|))+/g);
+
+	const id         = _.remove(tags, (tag)=>tag.startsWith('#')).map((tag)=>tag.slice(1))[0];
+	const classes    = _.remove(tags, (tag)=>(!tag.includes(':')) && (!tag.includes('=')));
+	let attributes = _.remove(tags, (tag)=>(!tag.includes(':')) && (!tag.includes('#')));
+	const styles     = tags?.length ? tags.map((tag)=>tag.replace(/:"?([^"]*)"?/g, ':$1;').trim()) : [];
 
-	if(!tags)	return '"';
+	if(attributes) {
+		attributes = attributes.map((attr)=>attr.replace(/="?([^"]*)"?/g, '="$1"'));
+	}
```

Comment:
I have an idea to simplify this slightly. I will push up a commit later after I make sure it works.

Reply:
Let me know. I tend to get sideswiped by a third party pushing a change at a PR from a personal fork. It's a weird pattern for me. :)

I banged my head on this particular bit a bunch and this was the only order of operations I found that worked.